language-server/modules/langserver-core/src/test/resources/completion/typedesc_context/config/function_typedesc17.json

```diff
@@ -15,39 +15,486 @@
       "insertTextFormat": "Snippet"
     },
     {
```

Comment:
This File Changed because

The code 
```
import ballerina/module1;

public const int TEST_CONST = 12;

public function main() {
    isolated f
    int[] testArr = [];
}

public type TestMap2 map<string>;

public type TestMap3 map<int>;

type TestMap1 map<int>;

type TestRecord1 record {
    int rec3Field1 = 12;
    string rec3Field2 = "TestRecord3";
};

```


Previously recovered as

```
import ballerina/module1;

public const int TEST_CONST = 12;

public function main() {
     INVALID[isolated] f
 MISSING[|]    int[] testArr = [];
}

public type TestMap2 map<string>;

public type TestMap3 map<int>;

type TestMap1 map<int>;

type TestRecord1 record {
    int rec3Field1 = 12;
    string rec3Field2 = "TestRecord3";
};
```

but Changes with this PR, It recovered as
```
import ballerina/module1;

public const int TEST_CONST = 12;

public function main() {
     INVALID[isolated] f
 MISSING[;]    int[] testArr = [];
}

public type TestMap2 map<string>;

public type TestMap3 map<int>;

type TestMap1 map<int>;

type TestRecord1 record {
    int rec3Field1 = 12;
    string rec3Field2 = "TestRecord3";
};

```